.github/workflows/regressions.json

```diff
@@ -4,7 +4,7 @@
         "projects": [
             {"name": "cargo", "repository": "rust-lang/cargo"},
             {"name": "tokio", "repository": "tokio-rs/tokio"},
-            {"name": "amethyst", "repository": "amethyst/amethyst"}
+            {"name": "nalgebra", "repository": "dimforge/nalgebra"}
```

Comment:
Does it really good replacement? It seems `amethyst` has a lot proc macros but `nalgebra` doesn't use them